packages/apps/dashboard/server/src/modules/details/details.spec.ts

```diff
@@ -0,0 +1,195 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { DetailsService } from './details.service';
+import { HttpService } from '@nestjs/axios';
+import { EnvironmentConfigService } from '../../common/config/env-config.service';
+import { NetworkConfigService } from '../../common/config/network-config.service';
+import { Logger } from '@nestjs/common';
+import { of, throwError } from 'rxjs';
+import {
+  ChainId,
+  ILeader,
+  OperatorUtils,
+  OrderDirection,
+} from '@human-protocol/sdk';
+import { LeadersOrderBy } from '../../common/enums/leader';
+
+jest.mock('@human-protocol/sdk', () => ({
+  ...jest.requireActual('@human-protocol/sdk'),
+  OperatorUtils: {
+    getLeaders: jest.fn(),
+  },
+}));
+
+const MOCK_MAX_LEADERS_COUNT = 5;
+
+jest.mock('../../common/constants/leader', () => ({
+  ...jest.requireActual('../../common/constants/leader'),
+  MAX_LEADERS_COUNT: MOCK_MAX_LEADERS_COUNT,
```

Comment:
By some reason we don't run tests for dashboard in CI 🤔 
This mock fails
<img width="717" alt="Screenshot 2024-12-18 at 17 32 58" src="https://github.com/user-attachments/assets/9f001339-b2f1-4df8-a52c-949bbaeb723b" />

Probably because of hoisting of `jest.mock` ([read this for ref](https://dev.to/jobber/serious-jest-making-sense-of-hoisting-253i))

There are some workarounds for it like e.g. using `var` instead of `const` and etc.